setup.py

```diff
@@ -66,12 +65,11 @@ def run_tests(self):
 # Extract name and e-mail ("Firstname Lastname <mail@example.org>")
 AUTHOR, EMAIL = re.match(r'(.*) <(.*)>', AUTHOR_EMAIL).groups()
 
-# For some versions of python (2.6) we need the importlib
-# as it is not in the standard install
-try:
-    import importlib
-except ImportError:
-    REQUIREMENTS.append('importlib')
+REQUIREMENTS = ['requests', 'dicttoxml']
+# Python 2.6 does not have importlib or OrderedDicts, so we need to install
+# the backports
+if sys.version_info < (2,7):
+    REQUIREMENTS.extend(['importlib', 'ordereddict'])
```

Comment:
If I don't remember wring, ordereddict was also one of the types in python-future, so yet another reason